recipe/meta.yaml

```
@@ -41,7 +41,8 @@ requirements:
   # See: https://github.com/man-group/ArcticDB/blob/master/environment_unix.yml
     - python
     - pip
-    - folly
+    # See reason for pin below
+    - folly <=2023.08.07.00
```

Now that the issue has been created, we can reference it here, I think.
```suggestion
    # See: https://github.com/man-group/ArcticDB/issues/765
    - folly <=2023.08.07.00
```